crates/driver/src/tests/setup/solver.rs

```diff
@@ -209,6 +219,31 @@ impl Solver {
             gas,
         )
         .await;
+        let deadline = time::Deadline::from(
+            config.deadline
+                // reduce by http delay to acoomodate for autopilot / driver communication
+                - chrono::Duration::milliseconds(
+                    default_http_time_buffer_milliseconds().try_into().unwrap(),
+                ),
+        );
+        let competition_time = match config.quote {
```

Comment:
I think config should have our new Deadline type from which it is simple to get the expected solver_duration.

Reply:
Yes, removed.